### DISCUSSION TITLE
Have the highlighting groups been renamed?

### DISCUSSION THREAD

**Discussion by hesxenon** [lukas-reineke/indent-blankline.nvim#838] [General]
I was trying to get colored indent guides and was met with an error that `RainbowRed` wasn't defined. Looking through the highlight groups I noticed that there is `RainbowDelimiterRed` and such... maybe lazy vim installed rainbow-delimiteris.nvim without directly showing it, or were the highlight groups renamed?

**Comment by lukas-reineke** [comment]
Neither `RainbowRed` nor `RainbowDelimiterRed` are, or were ever, part of indent-blankline.

`RainbowDelimiterRed` is from rainbow-delimiteris.nvim
and `RainbowRed` is just [an example in the readme](https://github.com/lukas-reineke/indent-blankline.nvim#multiple-indent-colors) of how you can define your own highlight groups.
